custom/icds_reports/ucr/data_sources/ccs_record_cases.json

```diff
@@ -594,6 +594,10 @@
               "type": "named",
               "name": "open_person_cases"
             },
+            {
+              "type": "named",
+              "name": "open_cases"
```

Comment:
Curious if both this and the existing `open_perscon_cases` filter are needed. What cases are caught by one and not the other?

Reply:
Abortion, where the ccs case gets closed is not being filtered out of the report correctly as earlier we were checking if only the person case is open or not @calellowitz

Reply:
Are there cases caught by `open_person_cases` that are not caught by `open_cases`? It is a bit simpler of a report if there is only one filter and it sounds like what we care about is whether the ccs_case is open, but if both are needed that is fine